src/FasterImage/FasterImage.php

```diff
@@ -68,6 +68,23 @@ class FasterImage
      */
     public function batch(array $urls)
     {
+        $has_curl_multi = (
```

Comment:
I know we have a record in github PR's, but I'd like to keep some of the knowledge in source too. Would be nice to have a comment here explaining why we're even checking this

```suggestion

        /**
         * It turns out that even when cURL is installed, the `curl_multi_init()
         * function may be disabled on some hosts who are seeking to guard against
         * DDoS attacks.
         *
         * @see https://github.com/ampproject/amp-wp/pull/2183#issuecomment-491506514.
         *
         * If it is disabled, we will batch these synchronously (with a significant
         * performance hit).
         */
        $has_curl_multi = (
```